src/var/intervalClear.js

```diff
@@ -0,0 +1,4 @@
+define(function() {
+	/* global clearInterval: false */
```

Comment:
I removed `clearInterval` from JSHint globals but this is the one place where it has to appear.